src/encoded/search.py

```diff
@@ -898,7 +899,54 @@ def set_filters(request, search, result, principals, doc_types):
     return search, final_filters
 
 
-def initialize_facets(request, doc_types, prepared_terms, schemas):
+def initialize_additional_facets(request, doc_types, additional_facets, append_facets, current_type_schema):
+    """ Helper function for below method that handles additional_facets URL param
+
+        :param request: current request
+        :param doc_types: doc_types we are searching on
+        :param additional_facets: extra facets specified
+        :param append_facets: list to add additional_facets to
+        :param current_type_schema: schema of the item we are faceting on
+    """
+    for extra_facet in additional_facets:
+        aggregation_type = 'terms'  # default
+
+        # check if defined in facets
+        if 'facets' in current_type_schema:
+            if extra_facet in current_type_schema['facets']:
+                if not current_type_schema['facets'][extra_facet].get('disabled', False):
```

Comment:
There's no advantage of specifying `False` here in place of the default `None` since you are in a boolean test context anyway. Missing from `.get(...)` is pretty well-accepted as false-y so the extra emphasis doesn't add much. Fine either way though.